clang/test/Driver/sycl-ftarget-register-alloc-mode.cpp

```diff
@@ -98,23 +98,23 @@
 
 // AUTO_AOT: clang-offload-packager{{.*}} "--image=file={{.*}}.bc,triple=spir64_gen-unknown-unknown,arch={{.*}},kind=sycl,compile-opts=-device_options [[DEVICE]] -ze-intel-enable-auto-large-GRF-mode{{.*}}"
 
-// LARGE_AOT: clang-offload-packager{{.*}} "--image=file={{.*}}.bc,triple=spir64_gen-unknown-unknown,arch=,kind=sycl,compile-opts=-device_options pvc -ze-opt-large-register-file"
+// LARGE_AOT: clang-offload-packager{{.*}} "--image=file={{.*}}.bc,triple=spir64_gen-unknown-unknown,arch=generic,kind=sycl,compile-opts=-device_options pvc -ze-opt-large-register-file"
```

Comment:
Why is 'change to arch' not an ABI breaking change?

Thanks

Reply:
The addition of `arch=generic` is separate to this change as well (not bundler).  Not considered an ABI breaking change as the new offloading model is not enabled.